components/Slider/README.md

```diff
@@ -40,6 +40,10 @@ or discrete set of values.
   - [Differences from UISlider](#differences-from-uislider)
 - [Extensions](#extensions)
   - [Color Theming](#color-theming)
+- [Accessibility](#accessibility)
+  - [Set `-accessibilityLabel`](#set-`-accessibilitylabel`)
```

Comment:
Nit: Let's standardize the headings to be just the property names. Please omit formatting from within the heading, since it I seems to break the anchors.